sdk/tests/client/node_api.rs

```diff
@@ -287,13 +287,17 @@ async fn test_get_milestone_by_id_raw() {
         .unwrap();
 
     let milestone = client.get_milestone_by_id(&latest_milestone_id).await.unwrap();
-    let milestone_raw = MilestonePayload::unpack_verified(
+    let milestone_raw = Payload::unpack_verified(
         client.get_milestone_by_id_raw(&latest_milestone_id).await.unwrap(),
         &client.get_protocol_parameters().await.unwrap(),
     )
     .unwrap();
 
-    assert_eq!(milestone, milestone_raw);
+    if let Payload::Milestone(milestone_raw) = milestone_raw {
```

Comment:
So one of these returns a `MilestonePayload` and one returns `Payload`? Why is that?

Reply:
Comes from a difference between the SDK and iota.go I guess, they don't have a payload enum so the type is on the variant

Reply:
I don't think we should modify the bytes returned by the API to remove the first one

Reply:
Alright 🤷‍♀️